mrmap/resourceNew/models/metadata.py

```diff
@@ -736,6 +737,12 @@ def remove_dataset_metadata_relation(self, related_object, relation_type, intern
             **kwargs
         ).delete()
 
+    def get_validate_url(self) -> str:
```

Comment:
i think it would be better to use `Conformity Check Run.get_add_url()` instead of implementing a code duplication which depends on other apps here.

Reply:
What I like about the approach of `get_validate_url` is that it allows us to define which resources support validation. For example a `MonitoringRun` should not support validation, I guess. How do we solve this by using the filter approach?

Reply:
I don't get your example with `MonitoringRun`. To validate the `CreateView` we could simply use form validation instead.